src/components/Icons/IconCalendarTime.jsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { string, number } from 'prop-types';
+
+const SvgComponent = ({ className, color, size }) => {
```

Comment:
Why is it called SvgComponent?

Reply:
should be file name

Reply:
oh mb, is my snippet that I forgot to rename, good catch! 😄